.github/workflows/audit.yml

```diff
@@ -25,3 +25,4 @@ jobs:
       - uses: rustsec/audit-check@v1.4.1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+          ignore: RUSTSEC-2020-0071
```

Comment:
This needs a comment why this was ignored, as well as a hint for future developers to know when they can unignore it